src/transformers/trainer.py

```diff
@@ -33,8 +33,19 @@
 from .training_args import TrainingArguments
 
 
-if is_apex_available():
-    from apex import amp
+_use_native_amp = False
+_use_apex = False
+
+# Check if Pytorch version >= 1.6 to switch between Native AMP and Apex
+if not version.parse(torch.__version__) >= version.parse("1.6"):
```

Comment:
Nit: can we remove the not and either use < in the comparison or switch the two parts inside the test? I find it hard to read this way.

Reply:
Thanks for the suggestion. This is resolved now.